.github/workflows/end-to-end.yml

```diff
@@ -56,6 +57,9 @@ jobs:
       - name: Run run workflow
         run: nf-test test --tag run --verbose
 
+      - name: Run run_dev_se workflow
```

Comment:
I would make a second github actions test, and call it end-to-end-se.yml, where you only test the run_dev_se. That way end-to-end.yml is standardized amongst different branches.

Reply:
But don't we want end-to-end.yml test everything? Would this be a temporary measure?

Reply:
resolved after discussion